bin/libs/common.sh

```diff
@@ -20,6 +20,22 @@ export _BPXK_AUTOCVT="ON"
 export _EDC_ADD_ERRNO2=1                        # show details on error
 unset ENV             # just in case, as it can cause unexpected output
 
+# Temporarily, the zwe init commands have old shell code and new jcl code paths available.
+# To determine which are used, we check if --generate is given (implies jcl)
+# If not, we check if zowe.setup.jcl.enabled is explicitly false.
+# If so, shell is used, otherwise jcl is used.
+check_jcl_init() {
+  enabled=$(yaml_read_configmgr "${ZWE_CLI_PARAMETER_CONFIG" ".zowe.setup.jcl.enable")
```

Comment:
The old code means one important limitation: config is only uss file.

We have to start this check in `check_configmgr_enabled` style:
https://github.com/zowe/zowe-install-packaging/blob/fe7030e0bdf6cd0615ab5c1934fef5cd21bae68e/bin/libs/common.sh#L31

* `--config /uss/path/zowe.yaml` 
  *  Old or new code depends on `zowe.setup.jcl.enable`
* `--config FILE(zowe.yaml):PARMLIB(ZOWE(YAML))`
  * New code automatically